Pipfile

```diff
@@ -0,0 +1,11 @@
+[[source]]
```

Comment:
Why do you need a top level Pipfile?

Reply:
I think it's because I was running pipenv from that folder. I can delete it